Themes/_fallback/Scripts/03 Gameplay.lua

```diff
@@ -329,7 +329,9 @@ local CodeDetectorCodes = {
 		pump = "",
 	},
 	CloseCurrentFolder = {
-		default = "MenuUp-MenuDown",
+		default = "",
```

Comment:
I think the default should not be empty. Let's continue using MenuUp-MenuDown as default?

Reply:
I agree with this